tests/benchmarks/fio_benchmark_test.py

```diff
@@ -153,5 +154,62 @@ def TestZeroMinutes(self):
     self.assertEquals(proc.call_count, 0)
 
 
+class TestFioTargetModeFlag(unittest.TestCase):
+  def doTargetModeTest(self, mode, fill_device=None, against_device=None,
```

Comment:
Consider renaming: fill_device to "expect_fill_device", against_device to "expect_against_device" and format_disk to "expect_format_disk"